icu4c/source/tools/ctestfw/unicode/ctest.h

```diff
@@ -89,7 +89,7 @@ U_CDECL_END
 #define WRITE_GOLDEN_DATA_OPTION 7
 
 /**
- * Maximum amount of memory uprv_malloc should allocate before returning NULL.
+ * Maximum amount of memory uprv_malloc should allocate before returning nullptr.
```

Comment:
Optional: The ctest.h functions are C functions, so NULL is appropriate. However, this is not library code...

Reply:
Done.